dotcom-rendering/src/web/components/ShowHideButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { css } from '@emotion/react';
+import { from, neutral, space, textSans } from '@guardian/source-foundations';
+import { ButtonLink } from '@guardian/source-react-components';
+
+type Props = {
+	sectionId: string;
+	overrideContainerToggleColour?: string;
+};
+
+const showHideButtonCss = (
+	overrideContainerToggleColour: string | undefined,
+) => css`
+	color: ${overrideContainerToggleColour ?? neutral[46]};
+	${textSans.xsmall()};
+
+	margin-top: ${space[2]}px;
+	margin-right: 10px;
+	margin-bottom: ${space[2]}px;
+	position: relative;
+	align-items: bottom;
+
+	${from.wide} {
+		position: absolute;
+		top: 0;
+		right: 0;
+	}
+`;
+
+export const ShowHideButton = ({
+	sectionId,
+	overrideContainerToggleColour,
+}: Props) => {
+	return (
+		<ButtonLink
+			priority="secondary"
+			subdued={true}
+			cssOverrides={showHideButtonCss(overrideContainerToggleColour)}
+			data-link-name="Hide"
+			data-show-hide-button={sectionId}
+			aria-controls={`container-${sectionId}`}
+			aria-expanded={true}
+		>
+			Hide
+		</ButtonLink>
+	);
```

Comment:
Could we add some kind of comment to this highlighting that the button functioanlity is added by `ShowHideContainers.importable`?

Something like
```
/**
 * This component creates the styled button for showing & hiding a container,
 * The functionality for this is implemented in a single island 'ShownHideContainers.importable'
 **/
```